server/download.go

```diff
@@ -94,7 +94,7 @@ func (p *blobDownloadPart) UnmarshalJSON(b []byte) error {
 }
 
 const (
-	numDownloadParts          = 64
+	numDownloadParts          = 4
```

Comment:
```suggestion
	numDownloadParts          = 16
```

Reply:
Context: from some testing this seemed to maintain a speed of 1GB/s on fast connections while still reducing the number of connections.